srslog_unix.go

```diff
@@ -34,16 +32,13 @@ func unixSyslog() (conn serverConn, err error) {
 // localConn adheres to the serverConn interface, allowing us to send syslog
 // messages to the local syslog daemon over a Unix domain socket.
 type localConn struct {
-	conn net.Conn
+	conn io.WriteCloser
 }
 
 // writeString formats syslog messages using time.Stamp instead of time.RFC3339,
 // and omits the hostname (because it is expected to be used locally).
-func (n *localConn) writeString(p Priority, hostname, tag, msg string) error {
-	timestamp := time.Now().Format(time.Stamp)
-	_, err := fmt.Fprintf(n.conn, "<%d>%s %s[%d]: %s",
-		p, timestamp,
-		tag, os.Getpid(), msg)
+func (n *localConn) writeString(framer Framer, f Formatter, p Priority, hostname, tag, msg string) error {
```

Comment:
This is related to the question about default parameters. `framer` and `formatter` are ignored by this function, even though, their interface looks to require them. Should this be signaled either by param names (which I don't believe go will allow) or with a comment?